plugins/inputs/procstat/procstat_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
Need to remove this build flag

Reply:
when I remove the build tag I get this error:
```
--- FAIL: TestGather_cgroupPIDs (0.00s)
        Error Trace:    procstat_test.go:363
        Error:          Received unexpected error:
                        open \sys\fs\cgroup\C:\Users\Vrecan\AppData\Local\Temp\565977507\cgroup.procs: The filename, directory name, or volume label syntax is incorrect.
```

Reply:
I addressed this by skipping the test in windows

Reply:
Not sure if it will be enough, but we should use filepath.Join here: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/procstat/procstat.go#L336